alcs-frontend/src/styles/functions.scss

```diff
@@ -0,0 +1,10 @@
+@use 'sass:math';
+
+@function rem($pixels, $context: 16) {
```

Comment:
The other decisions component uses the portal `rem` function. Is there any reason we can't use that instead?